sentences/fr-CA/fan_HassTurnOn.yaml

```diff
@@ -3,11 +3,11 @@ intents:
   HassTurnOn:
     data:
       - sentences:
-          - "<allume> ([tous] | [toutes] ) (<ventilateur> | <ventilateurs>) [<dans>] [<area>]"
+          - "<allume> ([tous]|[toutes]) (<ventilateur>|<ventilateurs>) [<dans>] [<area>]"
```

Comment:
and here, you don't have spaces around brackets :) That is right. But you are inconsistent with files :)